packages/simple-markdown/flow-typed/npm/mocha_v6.x.x.js

```diff
@@ -0,0 +1,226 @@
+// flow-typed signature: 03fc62001bb373473e2e4208d31a9f73
```

Comment:
I wonder how hard it would be to move all mocha tests to jest.  I think it'd be worthwhile.